### PR TITLE
[build] update gradle-errorprone-plugin to 1.1.0

### PR DESCRIPTION
**Pull request by grimreaper**
Problem
The plugin for error-prone is out of date

Solution
Update the plugin.

 - [x] Read the [contributing guide](https://github.com/mockito/mockito/blob/release/3.x/.github/CONTRIBUTING.md)
 - [x] PR should be motivated, i.e. what does it fix, why, and if relevant how
 - [x] If possible / relevant include an example in the description, that could help all readers
       including project members to get a better picture of the change
 - [x] Avoid other runtime dependencies
 - [x] Meaningful commit history ; intention is important please rebase your commit history so that each
       commit is meaningful and help the people that will explore a change in 2 years
 - [x] The pull request follows coding style
 - [x] Mention `Fixes #<issue number>` in the description _if relevant_
 - [x] At least one commit should mention `Fixes #<issue number>` _if relevant_

### CODE REVIEW

**Comment by TimvdLippe** [comment]
We should probably first update our Travis testing matrix to remove testing on Java 9 and 10, and adding Java 14. Then we can look into the failures with ErrorProne on Java 12. I will also contact the ErrorProne team to get them look at the problems.

**Comment by grimreaper** [comment]
agreed: though this is unrelated. I just tried this before realizing the problem was java 12 :)

**Comment by TimvdLippe** [comment]
Apparently the JDK 12 issue is being tracked at https://github.com/google/error-prone/issues/1106

**Comment by TimvdLippe** [comment]
Upstream has fixed their support for newer versions of Java. @grimreaper could you rebase this PR?

**Comment by grimreaper** [comment]
done

File: gradle/errorprone.gradle [hunk 1]
```diff
@@ -10,3 +10,7 @@ if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
 dependencies {
     errorprone libraries.errorprone
 }
+
+tasks.named("compileTestJava").configure {
+    options.errorprone.errorproneArgs.add("-Xep:MockitoCast:OFF")
```
Comment (TimvdLippe) [inline gradle/errorprone.gradle:15]: This should be removed, as this check no longer exists.

Comment (grimreaper) [reply gradle/errorprone.gradle]: bad merge resolve. fixed.

**Pull request merged**